RELEASE.md

```diff
@@ -103,11 +103,20 @@ Entries in the `CHANGELOG.md` are meant to be in this order:
 Tag the new release via the following commands:
 
 ```bash
-$ tag=$(< VERSION)
-$ git tag -s "v${tag}" -m "v${tag}"
-$ git push origin "v${tag}"
+$ tag="v$(< VERSION))"
+$ git tag -s "${tag}" -m "${tag}"
+$ git push origin "${tag}"
 ```
 
+Optionally, you can use this handy `.gitconfig` alias.
```

Comment:
I'm not sure about asking people to adjust their personal config files like this, if they want to this they already can.

Reply:
It's marked as optional and is not required for the workflow, which seems fine to me.

Reply:
We should make the alias read VERSION then

Reply:
I wasn't sure how much shell functionality I could assume in git aliases. Doing some tests, it looks somewhat limited.

This works:
```
tag-release = "!f() { v=${1:-$(cat VERSION)} ; echo git tag -s v$v -m v$v && echo git push origin v$v; }; f"
```

But this fails:
```
tag-release = "!f() { v=${1:-$(< VERSION)} ; echo git tag -s v$v -m v$v && echo git push origin v$v; }; f"
```